app/controllers/candidate_interface/rejection_feedback_survey_controller.rb

```diff
@@ -0,0 +1,19 @@
+module CandidateInterface
+  class RejectionFeedbackSurveyController < CandidateInterfaceController
+    def new
+      ProvideRejectionFeedback.new(application_choice_params, helpful_params).call
+      redirect_to candidate_interface_application_complete_path
+      flash[:success] = 'Feedback successfully provided'
```

Comment:
is the overloading of the word "feedback" a bit confusing here? we've asked "is this feedback helpful?" and now we're telling them "feedback successfully provided". perhaps something along the lines of "thanks, your response has been recorded".

Reply:
Hopefully we'll be able to remove this flash message altogether (for most users) by switching up to an AJAX submission in future...